src/main/java/Charlie.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
```

Comment:
It would be good to add a Javadoc comment for the class, as well as the public methods inside the class.